pkg/trace/agent/agent.go

```diff
@@ -258,6 +270,7 @@ func (a *Agent) Process(t pb.Trace) {
 
 	go func(pt ProcessedTrace) {
 		defer watchdog.LogOnPanic()
+		defer a.timer.Time("datadog.trace_agent.internal.concentrator_ms", time.Now())
```

Comment:
Do we want `time.Now()` to be evaluated before this anonymous function actually starts running? It's probably a minor difference, but it would be clearer I think to measure all completion of all 3 anonymous functions from the same start time?

Reply:
Do you know what else happens between before the `go func` declaration and until it starts? Perhaps there's other runtime things involved, scheduler waits, who knows? I don't. I was thinking this would be more accurate and reflect the timing strictly for our own code.